tests/performance/install-local/task.yaml

```diff
@@ -0,0 +1,12 @@
+summary: Measure the time needed to install a local snap
+
+execute: |
+    . "$TESTSLIB/snaps.sh"
+    . "$TESTSLIB/perf.sh"
+
+    for i in $(seq 30); do
+        et=$(execution_time_ms "install_local test-snapd-tools")
```

Comment:
Doesn't install_local also build the snap?

Reply:
BTW. makes me wonder if there's a huge different between the data reported by `snap changes` and measuring from the outside.